### ISSUE TITLE
Some Jupyter Icons gone

### ISSUE THREAD

**Issue by echyam** [jupyterlab/jupyterlab#7129]
## Description

Some images/icons (filebrowser, running terminals, command palette, open tabs) are failing to load and display a giant red box (in Chrome) or blank white space (in Firefox).  interestingly enough, non-jupyter native icons load fine (e.g. Git icons are fine)

## Reproduce
I don't know how to describe how to reproduce this, since it doesn't respond to any changes I make.  This is what I see with Chrome:
![Screen Shot 2019-08-29 at 6 51 21 PM](https://user-images.githubusercontent.com/11476825/63987679-0637d580-ca8e-11e9-81f6-680bee51e4af.png)

If I switch the filebrowser to the right side, I can see more of the box:
![Screen Shot 2019-08-29 at 6 51 48 PM](https://user-images.githubusercontent.com/11476825/63987724-29fb1b80-ca8e-11e9-9725-d84621142818.png)

This is what I see when I use Firefox (only have the Git and Terminals Tab):
![Screen Shot 2019-08-29 at 6 55 04 PM](https://user-images.githubusercontent.com/11476825/63987870-89592b80-ca8e-11e9-8faa-d0f91f9ef31e.png)

## Expected behavior

Expect normal icons to appear when loading workspace. 

## Context

- Operating System and version: Debian 9.5 (stretch)
- Browser and version: 
	- Chrome: Version 76.0.3809.132 (Official Build) (64-bit)
	- Firefox: 52.8.1 (64-bit)
- JupyterLab version: 1.0.2 (jupyter version 4.4.0)

<details><summary>Troubleshoot Output</summary>
<pre>
Paste the output from running `jupyter troubleshoot` from the command line here.
You may want to sanitize the paths in the output.
</pre>
</details>

<details><summary>Command Line Output</summary>
<pre>
Nothing noticeable here
</pre>
</details>

<details><summary>Browser Output</summary>
<pre>
react.development.js:188 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `NotebookTrustComponent`.
    in NotebookTrustComponent
    in div
warningWithoutStack @ react.development.js:188
warning @ react.development.js:603
createElementWithValidation @ react.development.js:1730
NotebookTrustComponent @ truststatus.js:37
renderWithHooks @ react-dom.development.js:12938
mountIndeterminateComponent @ react-dom.development.js:15020
beginWork @ react-dom.development.js:15625
performUnitOfWork @ react-dom.development.js:19312
workLoop @ react-dom.development.js:19352
callCallback @ react-dom.development.js:149
invokeGuardedCallbackDev @ react-dom.development.js:199
invokeGuardedCallback @ react-dom.development.js:256
replayUnitOfWork @ react-dom.development.js:18578
renderRoot @ react-dom.development.js:19468
performWorkOnRoot @ react-dom.development.js:20342
performWork @ react-dom.development.js:20254
performSyncWork @ react-dom.development.js:20228
requestWork @ react-dom.development.js:20097
scheduleWork @ react-dom.development.js:19911
scheduleRootUpdate @ react-dom.development.js:20572
updateContainerAtExpirationTime @ react-dom.development.js:20600
updateContainer @ react-dom.development.js:20657
push.Ybsr.ReactRoot.render @ react-dom.development.js:20953
legacyRenderSubtreeIntoContainer @ react-dom.development.js:21105
render @ react-dom.development.js:21155
(anonymous) @ vdom.js:60
renderDOM @ vdom.js:52
onUpdateRequest @ vdom.js:30
push.FmDU.Widget.processMessage @ widget.js:492
invokeHandler @ index.js:436
sendMessage @ index.js:172
runMessageLoop @ index.js:486
requestAnimationFrame (async)
enqueueMessage @ index.js:455
postMessage @ index.js:223
push.FmDU.Widget.update @ widget.js:354
push.eIPM.CommandPalette.refresh @ commandpalette.js:196
push.eIPM.CommandPalette._onGenericChange @ commandpalette.js:434
invokeSlot @ index.js:475
emit @ index.js:433
push.qUp9.Signal.emit @ index.js:106
push.5TpB.CommandRegistry.addCommand @ index.js:126
addCommands @ index.js:386
activate @ index.js:48
(anonymous) @ index.js:147
Promise.then (async)
push.2R+v.Application.activatePlugin @ index.js:146
(anonymous) @ index.js:266
push.2R+v.Application.start @ index.js:265
main @ index.out.js:1833
load (async)
ANye @ index.out.js:1879
__webpack_require__ @ bootstrap:84
0 @ bootstrap:221
__webpack_require__ @ bootstrap:84
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ vendors~main.76b5f876be8046842c43.js:1
Show 2 more frames
react-dom.development.js:55 Uncaught Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `NotebookTrustComponent`.
    at invariant (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:423515:15)
    at createFiberFromTypeAndProps (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:433677:11)
    at createFiberFromElement (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:433698:15)
    at reconcileSingleElement (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:435991:23)
    at reconcileChildFibers (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:436048:35)
    at reconcileChildren (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:437862:28)
    at mountIndeterminateComponent (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:438524:5)
    at beginWork (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:439085:16)
    at performUnitOfWork (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:442772:12)
    at workLoop (https://che-k8s.maap.xyz/servercls245hi-ws-jupyter/server-3100/static/lab/vendors~main.76b5f876be8046842c43.js:442812:24)
</pre>
</details>

**Comment by telamonian** [comment]
1. Try upgrading to the latest version of Jupyterlab, `v1.1.1`. Some possibly relevant code changed between your version and the new one.

2. If that doesn't work, please post here the text contents of one of those red warning boxes that you're seeing. Please also post some information about the system you're running on, especially the OS.

**Comment by echyam** [comment]
1. I'll try upgrading to `v1.1.1`
2. It's running inside a docker container with base Debian 9.5 (stretch)

**Comment by ieglonew01f** [comment]
https://github.com/jupyterlab/jupyterlab/commit/8cb310869314b6684f6ae2386381d1dfb2956664 is the likely culprit here, badly handled js objects.

**Comment by jasongrout** [comment]
> [8cb3108](https://github.com/jupyterlab/jupyterlab/commit/8cb310869314b6684f6ae2386381d1dfb2956664) is the likely culprit here, badly handled js objects.

Probably not, since that code wasn't in 1.0.2, just in 1.1.0. The OP said they were running 1.0.2, and were going to upgrade and test on 1.1.1.

**Comment by ginsy3000** [comment]
Have this same issue. When I try to update it just keeps me at 1.0.2. Using the guide on this page https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html

**Comment by jasongrout** [comment]
That guide is for new installation. To update, I think you either need to use your package manager update flag, or just give the explicit version to the install command, like `pip install jupyterlab=1.1.1` or something.

**Comment by danieljunior** [comment]
> That guide is for new installation. To update, I think you either need to use your package manager update flag, or just give the explicit version to the install command, like `pip install jupyterlab=1.1.1` or something.

just a misspelling: `pip install jupyterlab==1.1.1`

**Comment by krassowski** [comment]
I had an issue with a missing icon today on 1.0.4/1.0.5. Everything worked fine until I installed the TOC extension and then in the build logs (watch mode) I saw that an SVG icon cannot be imported from notebook anymore. There was something about statusbar too (I remember that there was base64 code instead of an icon there), I don't have these logs anymore and do not remember exactly.

I remember seeing the same glitch with the filebrowser icons as on the first screenshot.
It all went away once I updated everything to 1.0.9.

Long story short: installation of some extensions may cause this issue.

I guess that some of the core packages has a backward incompatible change in 1.0.x or that some extensions are pulling 1.1.x core packages, but I do not understand why any of those would lead to a conflict (maybe something with the build system preferring to include the newer version when rebuilding after extension installation - IDK, just a guess I may be totally wrong).

**Comment by jasongrout** [comment]
@echyam - did upgrading to 1.1.1 fix things for you?

**Comment by ieglonew01f** [comment]
@jasongrout It worked for me thanks !

**Comment by 20v100** [comment]
I had the same issue than @krassowski. `pip install jupyterlab==1.1.1` solved the issue.

**Comment by echyam** [comment]
@jasongrout sorry, I was out for the long weekend.  
Upgrading to 1.1.1 worked.  Thanks!
